src/main/java/com/google/graphgeckos/dashboard/datatypes/Builder.java

```diff
@@ -12,10 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.graphgeckos.dashboard.storage;
+package com.google.graphgeckos.dashboard.datatypes;
 
 import java.util.ArrayList;
 import java.util.List;
+
```

Comment:
please keep all (non-static) imports in a single block: https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

Reply:
Done!